test/upgrade/prober/prober.go

```diff
@@ -34,20 +33,16 @@ var (
 )
 
 // Prober is the interface for a prober, which checks the result of the probes when stopped.
-type Prober interface {
-	// Verify will verify prober state after finished has been send
-	Verify() ([]error, int)
-
-	// Finish send finished event
-	Finish(ctx context.Context)
-
-	// ReportErrors will reports found errors in proper way
-	ReportErrors(t *testing.T, errors []error)
+type Prober struct {
```

Comment:
perhaps the comment above needs to be updated as the Prober no longer performs the check?

Reply:
Done